### DISCUSSION TITLE
[Feature Request] Windows 10 new search icon

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1701] [Ideas]
![屏幕截图 2023-07-28 113034](https://github.com/valinet/ExplorerPatcher/assets/140480110/a650a5df-ce41-4443-afa7-52a21f69dbf9)

**Comment by XuanChen1234** [comment]
Any more planned?

**Comment by milestprower92** [comment]
> ![屏幕截图 2023-07-28 113034](https://private-user-images.githubusercontent.com/140480110/256716574-a650a5df-ce41-4443-afa7-52a21f69dbf9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMTg1NjYsIm5iZiI6MTcwMjMxODI2NiwicGF0aCI6Ii8xNDA0ODAxMTAvMjU2NzE2NTc0LWE2NTBhNWRmLWNlNDEtNDQ0My1hZmE3LTUyYTIxZjY5ZGJmOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMVQxODExMDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01MWNkYTQ0NjQ4OTdlNDJjNzU2OGYxMWVlYzI0NDgwYzEyMzQ2MjAzMTVmMWE0NDU5OTg0OTU5MGVjNmU2ZjJlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.nQMNzsWE1sgrMgmvgfqRBifpzeBTXQQFy1XN4yVAGYk)

Never
